Update README.md

inkade opened the pull request:
Bulked out README

Honry commented on README.md:
```diff
@@ -2,24 +2,67 @@
+2. Navigate to the desired sample directory that you want to explore.
+3. Read the accompanying README.md file for the sample to understand its purpose, requirements, and implementation details.
+4. Follow the instructions provided in the README to set up the necessary dependencies and run the sample.
+5. Experiment with the code and explore how WebNN can enhance machine learning tasks in the browser, navigating to http://localhost:8080.
```
We may need to emphase currently WebNN implementation only supports CPU backend on Chrome or Edge behind a flag: [#enable-experimental-web-platform-features](chrome://flags/#enable-experimental-web-platform-features)

ibelem replied:
@Honry, I think @inkade may have no idea on this, we can add it by ourselves.

Honry reviewed (commented):
@inkade, awesome, thanks much for your contribution! That's really helpful!

Honry commented:
@ibelem, PTAL as well. Thanks!

Honry commented on README.md:
```diff
@@ -2,24 +2,67 @@
   * [RNNoise](https://webmachinelearning.github.io/webnn-samples/rnnoise/)
 * [Fast style transfer](https://webmachinelearning.github.io/webnn-samples/style_transfer/)
-* [Image classification](https://webmachinelearning.github.io/webnn-samples/image_classification/)
-* [Object detection](https://webmachinelearning.github.io/webnn-samples/object_detection/)
```
Mistake? Don't remove these two lines.

inkade commented:
Thanks for the feedback guys, I appreciate it.

I have updated with the following:
* Switched to relative paths for 'Repository Structure' URLs.
* Emphasised the compatible browser requirement by placing the 'Requirements' section before 'Getting Started'.
* Provided clarity and guidance regarding enabling the #enable-experimental-web-platform-features flag.
* Reintergrated 'Image classification' and 'Object detection' links within 'WebNN API Samples'.

Regards

Honry reviewed (approved):
Pretty good! Thanks!
